indicator/tasks.py

```diff
@@ -36,7 +36,7 @@
 
 
 @celery_app.task(bind=True, name=_("Generate scientific indicator"))
-def task_generate_article_indicators(self, user_id, indicators):
+def task_generate_article_indicators(self, user_id, indicators, remove=False):
```

Comment:
@gitnnolabs acho que há vários pontos de mudanças
- é uma só task para gerar todos os indicadores, sem possibilidade de filtro, sem poder gerar um indicador específico, sempre que for reprocessar, só é possível reprocessar tudo? sugiro haver uma task que a partir de parâmetros gera 1 indicador; uma task para executar todos os indicadores;
- sugiro uma task só para gerar os dados crus de um dado registro de indicador
-